### PR TITLE
HomeScreen updates

### PR DESCRIPTION
**Pull request by lucasvanhalst**
* update plugin to support new home screen in app
* adds rogerthat.getBadges

TODO: iOS version of rogerthat.getBadges

### CODE REVIEW

File: src/android/RogerthatPlugin.java [hunk 1]
```diff
@@ -200,6 +204,9 @@ private void processAction(String action, CallbackContext callbackContext, JSONO
             case "context":
                 getContext(callbackContext);
                 break;
+            case "getBadges":
```
Comment (RubenMattan) [inline src/android/RogerthatPlugin.java]: lets refactor to badges_list

File: www/RogerthatPlugin.js [hunk 2]
```diff
@@ -293,10 +293,12 @@ RogerthatPlugin.prototype.features = {
     callback: undefined
 };
 
-RogerthatPlugin.prototype.getBadges = function(){
-    return new Promise(function(resolve, reject){
-        utils.exec(resolve, reject, "getBadges", []);
-    });
+RogerthatPlugin.prototype.badges = {
```
Comment (lucasvanhalst) [inline www/RogerthatPlugin.js]: needs to stay getBadges otherwise you have to modify 3 or 4 embedded apps that are already made atm

Comment (RubenMattan) [reply www/RogerthatPlugin.js]: then update those embedded apps

Comment (lucasvanhalst) [reply www/RogerthatPlugin.js]: why? rogerthat.getBadges is simpler and there's no other badges functionality

**Pull request merged**